docs/release.md

```
@@ -1,6 +1,32 @@
+opening PRs for updating the cargo toml and lock files, pinning Rust
+version to use and updating the CHANGELOG.
+
+Once a release engineer signals a new tag for fact is needed, the
+`Release New Version` workflow can be used to push this tag
+automatically. It will work for both the initial version of a new X.Y
+stream, as well as for any future patch releases needed for that same
+stream.
+
+## Manual release
+
+**Note**: This is a fallback method, only intended to be used if the
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)